tools/run_pr_builder.sh

```diff
@@ -18,7 +18,17 @@ node --version
 # go list is expensive, only do it once.
 GO_LIST="$(go list ./... | grep -v /vendor/ | grep -v /samples/js/)"
 go build ${GO_LIST}
-go test ${GO_LIST}
+
+# go test plus build coverage data for upload codecov.io
+echo "" > coverage.txt
+for d in ${GO_LIST}; do
+    go test -coverprofile=profile.out -covermode=atomic $d
+    if [ -f profile.out ]; then
+        cat profile.out >> coverage.txt
+        rm profile.out
+    fi
+done
+bash <(curl -s https://codecov.io/bash) -t ${COVERALLS_TOKEN}
```

Comment:
I could, but then we won't get updates as they change and improve the script for all of the various languages and code coverage tools. Somewhat ironically this is like the Google Chrome auto-updates arguments. I did have the same gut instinct as you on this, but they do have information on the codecov.io site that helped allay my fears. That plus it is open source, so it is something that gets reviewed. I am mildly against checking in the script. If you feel strongly let me know and I will do it.